src/components/CCPDashboard/PatientInfoTable.tsx

```diff
@@ -155,9 +161,22 @@ export const PatientInfoTable = ({
     patients.find((x) => x.id === patientId)
   );
   const [anchorEl, setAnchorEl] = React.useState(null);
+  const [runNumber, setRunNumber] = React.useState<number | null>(
+    selectedPatient ? selectedPatient.runNumber : null
+  );
+  const [isNewRunNumber, setIsNewRunNumber] = React.useState<boolean>(false);
```

Comment:
is there a way to eliminate this variable? we already have `runNumber`, can't we just check that wherever we currently look at `isNewRunNumber` (i.e. just compare the saved run number with the one that's currently in the input field)?

Reply:
Yeah, that makes sense. Thanks, I've made the change in the latest commit.